nixos/modules/services/audio/jmusicbot.nix

```
@@ -27,7 +34,7 @@ in
       after = [ "network-online.target" ];
       description = "Discord music bot that's easy to set up and run yourself!";
       serviceConfig = mkMerge [{
-        ExecStart = "${pkgs.jmusicbot}/bin/JMusicBot";
+        ExecStart = "${package}/bin/JMusicBot";
```

```suggestion
        ExecStart = "${cfg.package}/bin/JMusicBot";
```

This change shouldn't work without this, right?

---

Thank you, you are right